avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -1251,7 +1282,7 @@ def computeGradC(cfg, particles, headerNeighbourGrid, headerNormalGrid, double[:
                   if r < rKernel:
                       hr = rKernel - r
                       dwdr = dfacKernel * hr * hr
-                      mdwdrr = mass[l] * (1 - h[j]/h[l]) * dwdr / r
+                      mdwdrr = mass[l] * (1 - hArray[k]/hArray[l]) * dwdr / r
```

Comment:
maybe also create hl and hk

Reply:
do we actually need both? If it is only for testing, then it should not go into the master...

Reply:
it's not in the option I modified, I do prefer modify it in a further PR